src/lib/uTop_utils.ml

```diff
@@ -0,0 +1,33 @@
+
+module Tailrec_List = struct
+  include List
+
+  let map f l =
+    List.rev (List.rev_map f l)
+
+  let map2 f l l' =
+    List.rev (List.rev_map2 f l l')
+
+  let concat l =
+    let rec aux acc = function
+    | [] -> acc
+    | h :: t ->
+       aux (List.rev_append h acc) t
+    in
+    List.rev (aux [] l)
```

Comment:
I'm wondering if we shouldn't just use a stdlib more systematically for community projects. That would make maintaining them easier overall